students/JeroenSweerts/Lesson3/Slicing_Lab_Exercise.py

```diff
@@ -7,19 +7,7 @@
 
```

Comment:
pep8: your python file names should be all lower case -> `slicing_lab_exercise.py`